src/Acuminator/Acuminator.Analyzers/StaticAnalysis/PXGraphCreationDuringInitialization/PXGraphCreationDuringInitializationAnalyzer.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Acuminator.Analyzers.StaticAnalysis.PXGraph;
+using Acuminator.Utilities.Roslyn;
+using Acuminator.Utilities.Roslyn.Semantic.PXGraph;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Collections.Immutable;
+
+namespace Acuminator.Analyzers.StaticAnalysis.PXGraphCreationDuringInitialization
+{
+    public class PXGraphCreationDuringInitializationAnalyzer : IPXGraphAnalyzer
+    {
+        public ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics =>
+            ImmutableArray.Create
+            (
+                Descriptors.PX1057_PXGraphCreationDuringInitialization
+            );
+
+        public void Analyze(SymbolAnalysisContext context, PXContext pxContext, PXGraphSemanticModel pxGraph)
+        {
+            context.CancellationToken.ThrowIfCancellationRequested();
+
+            if (pxGraph.Initializers == null)
```

Comment:
@SENya1990 I agree about null check. I'll remove it. No null check for pxContext and pxGraph because this method is intended to be used only by us and in this case they won't be null